packages/megabasterd-bin/megabasterd-bin.pacscript

```diff
@@ -0,0 +1,38 @@
+name="megabasterd-bin"
+gives="megabasterd"
+pkgver="8.21"
+pkgdesc="Yet another unofficial (and ugly) cross-platform MEGA downloader/uploader/streaming suite"
+homepage="https://github.com/tonikelope/${gives}"
+depends=("java11-runtime")
```

Comment:
Does it only work on java11?

Reply:
Java11 or later. But some of the GitHub runners were pulling Java8 to satisfy the `java-runtime` dependency